tests/Fixtures.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+#include "UsgsAstroFramePlugin.h"
+#include "UsgsAstroFrameSensorModel.h"
+
+#include <json.hpp>
+
+#include <sstream>
+#include <fstream>
+
+#include <gtest/gtest.h>
+
+using json = nlohmann::json;
+
+// Should this be positioned somehwere in the public API?
+inline void jsonToIsd(json &object, csm::Isd &isd, std::string prefix="") {
+  for (json::iterator it = object.begin(); it != object.end(); ++it) {
+     json jsonValue = it.value();
+
+     // Is this type of type sensative de-serialization not available
+     // out of the box?
```

Comment:
I think we can define some stuff to make this work more elegantly, the JSON library is rather complex because of how much it does. I've tried this before and it was a huge pain debugging it.